e2e/pages/adf/settingsPage.ts

```diff
@@ -96,9 +98,43 @@ export class SettingsPage {
         return this;
     }
 
-    clickApply() {
+    async clickSsoRadioButton () {
+        Util.waitUntilElementIsVisible(this.ssoRadioButton);
+        await this.ssoRadioButton.click();
+    }
+
+    async setSSO (processServiceURL, authHost ) {
+        this.goToSettingsPage();
+        this.setProvider(this.bpm.option, this.bpm.text);
+        Util.waitUntilElementIsVisible(this.bpmText);
+        expect(this.ecmText.isPresent()).toBe(false);
```

Comment:
This expect could be changed with Util.waitUntilElementIsNotOnPage(this.ecmText);